python/src/python.cpp

```diff
@@ -383,13 +378,16 @@ XPORT(int, MakeDirectory)(HANDLE hPlugin,const wchar_t **Name,int OpMode) {
     PYTHON_INT(0)
     return result;
 }
+#if 0
+// in far2l only one plugin is allowed at time
```

Comment:
what was wrong with this?

Reply:
Currently OpenFilePlugin uses the plugin manager itself, not the plugins installed with it.
It's the same with the analyze api

I still have no idea how to use this api, so I turned it off.

Reply:
Yep its same as AnalyseW, but only difference that later is FAR3-specific.
A alternative usage example can tell that NetRocks uses it to 'open' special ini file that describes connection.

Reply:
For now, I will finish this plugin, in a moment there will be another pull anyway - adb is much faster :)